lua/chatcommands/coinflip.lua

```diff
@@ -0,0 +1,102 @@
+CoinFlips = CoinFlips or {}
+RegisterChatCommand({'coin','coinflip'},function(ply,arg)
+	if arg:lower() == "accept" then checkCoinFlipRequest(ply) return end
```

Comment:
Honestly, this design won't work. Say person A makes a request to B to flip for 1000 points.  Then while person B is typing "!coin accept", malicious user C types "!coin B 1000000" and suddenly B loses 1 million. This is 100% something that would happen over and over.

I suggest make the other player type `!coin accept <amount>` to confirm the bet amount

Reply:
I get what you are saying. What about instead of that only 1 request can be made to a person at a time?

Reply:
then someone can lock it, so no

Reply:
Honestly I don't think someone is going to be attempting to lock someone out of a coinflip for 60 seconds. I just think that "!coinflip accept amount" is such a weird command. How about if they only have 1 request they can do "!coinflip accept" but if they have multiple requests they have to do "!coinflip accept amount"

Reply:
Honestly I think you still need to make people type the amount. I don't want people asking me for refunds because "I thought the flip was only for 10k not 100k" and whatnot. It would happen a lot.